packages/hub/model.js

```diff
@@ -27,6 +27,13 @@ module.exports = class Model {
     }
   }
 
+  // TODO watch out its probblyok for a relationship to have both links and data...
+  getLinks(fieldName) {
```

Comment:
Instead of adding this method, we should change `getField` so that for relationships it returns the whole `{ data, links }`.

Reply:
I like this idea, but `getField` already returns `relationships.${fieldName}.data` for relationships, and I would assume there is code that relies on this behavior. Hopefully it shouldn't be too large of a refactor.